Stellar-transaction.x

```diff
@@ -775,11 +775,18 @@ case PRECOND_V2:
     PreconditionsV2 v2;
 };
 
+// If bumpLedgers is not empty, the given key's lifetime is increased by bumpLedgers
+struct FootprintEntry
+{
+    LedgerKey key;
+    uint32* bumpLedgers;
```

Comment:
I'm not a huge fan of special entries, but here it seems to me like setting `bumpLedgers` to 0 by default would have the right semantics and save 1-4 bytes per footprint entry (not sure what is the optional field overhead).